mingus/midi/midi_file_in.py

```diff
@@ -67,7 +67,15 @@ class MidiFile(object):
 
     def MIDI_to_Composition(self, file):
         (header, track_data) = self.parse_midi_file(file)
+#        print('MIDI_to_Composition:', track_data[1])
+#        print('===')
+#        print(track_data[2])
+#        for j in track_data[2]:
+#            if j[1]['event'] == 9 or j[1]['event'] == 8:
+#                print('MIDI_to_Composition:', j[0], j[1]['event'], Note(notes.int_to_note(j[1]['param1'] % 12))) 
```

Comment:
Commented out code should not be committed. Seems like a debug thing in this case.